README.md

```diff
@@ -126,47 +88,84 @@ kustomize build 'github.com/kubernetes-sigs/descheduler/kubernetes/deployment?re
 
 See the [user guide](docs/user-guide.md) in the `/docs` directory.
 
-## Policy and Strategies
+## Policy, Default Evictor and Strategy plugins
+
+**⚠️ v1alpha1 configuration is still suported, but deprecated (and soon will be removed). Please consider migrating to v1alpha2 (described bellow). For previous v1alpha1 documentation go to docs/deprecated/v1alpha1.md**
+
+The Descheduler Policy is configurable and includes default eviction strategy plugins that can be enabled or disabled. It includes a common eviction configuration at the top level, as well as configuration from the Evictor plugin (Default Evictor, if not specified otherwise). Top-level configuration and Evictor plugin configuration are applied to all evictions.
 
-Descheduler's policy is configurable and includes strategies that can be enabled or disabled. By default, all strategies are enabled.
+### Top Level configuration
+
+These are top level keys in the Descheduler Policy that you can use to configure all evictions.
+
+| Name | Default Value | Description |
+|------|---------------|-------------|
+| `nodeSelector` | `nil` | limiting the nodes which are processed |
+| `maxNoOfPodsToEvictPerNode` | `nil` | maximum number of pods evicted from each node (summed through all strategies) |
+| `maxNoOfPodsToEvictPerNamespace` | `nil` | maximum number of pods evicted from each namespace (summed through all strategies) |
+
+### Evictor Plugion configuration (Default Evictor)
+
+The Default Evictor Plugin is used by default for filtering pods before processing them in an eviction strategy plugin, or for applying a PreEvictionFilter of pods before eviction. You can also create your own Evictor Plugin or use the Default one provided by Descheduler.  Other uses for the Evictor plugin can be to sort, filter, validate or group pods by different criteria, and that's why this is handled by a plugin and not configured in the top level config.
 
-The policy includes a common configuration that applies to all the strategies:
 | Name | Default Value | Description |
 |------|---------------|-------------|
 | `nodeSelector` | `nil` | limiting the nodes which are processed |
```

Comment:
Worth mentioning the param is used only when `nodeFit=true` and only by the `PreEvictionFilter` EP

Reply:
:heavy_check_mark: